go/main.go

```diff
@@ -111,18 +122,26 @@ func main() {
 		},
 		Reporter: &jaegercfg.ReporterConfig{
 			CollectorEndpoint: "http://jaeger:14268/api/traces",
-			LogSpans:          false,
+			LogSpans:          true,
 		},
 	}
 
+	jLogger := jaegerlog.StdLogger
+	jMetricsFactory := metrics.NullFactory
+	zipkinPropagator := zipkin.NewZipkinB3HTTPHeaderPropagator()
+
 	closer, err := cfg.InitGlobalTracer(
 		"example/server/go",
-		jaegercfg.Logger(jaegerlog.StdLogger),
+		jaegercfg.Logger(jLogger),
+		jaegercfg.Metrics(jMetricsFactory),
+		jaegercfg.Injector(opentracing.HTTPHeaders, zipkinPropagator),
+		jaegercfg.Extractor(opentracing.HTTPHeaders, zipkinPropagator),
+		jaegercfg.ZipkinSharedRPCSpan(true),
```

Comment:
This is the primary change, switching to the zipkin tracer and using the shared span as documented